test/mocks/src/user.mock.js

```diff
@@ -110,6 +114,11 @@ export default class UserMock extends User {
     nock(this.client.apiHostname, { encodedQueryParams: true })
       .post(`${this.pathname}/_logout`)
       .reply(204);
+      
+    if (LiveServiceFacade.isInitialized()) {
```

Comment:
Is the `user.logout()` behavior covered anywhere in tests? I may have missed it, but it'll be useful to test the state of the live service instances when a user is logged out.